src/Summoner/Template.hs

```diff
@@ -81,8 +81,8 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
         version:             0.0.0
         description:         $description
         synopsis:            $description
-        homepage:            https://github.com/${owner}/${repo}
-        bug-reports:         https://github.com/${owner}/${repo}/issues
+        $githubHomepage
+        $githubBugReports
```

Comment:
Maybe do it as `readme` recently.

Reply:
Yes, it's possible. But `.cabal` file is much bigger. I think it's a minor inconvenience and we can live with that right now. It's still good to have some interpolation quasi-quoter instead of manually specifying each file as list of lines.